src/logic/documents/scorecards.js

```diff
@@ -265,6 +266,48 @@ const scorecards = (wcif, rounds, rooms, language) => {
   return cards;
 };
 
+const shouldPrintScrambleChecker = (competitor, round, activityCode, wcif) => {
+  const { eventId } = activityCode ? parseActivityCode(activityCode) : {};
+  const {
+    printScrambleCheckerForTopRankedCompetitors,
+    printScrambleCheckerForFinalRounds,
+    printScrambleCheckerForBlankScorecards,
+  } = getExtensionData('CompetitionConfig', wcif);
+
+  let printScrambleCheckerBox = false;
+  if (printScrambleCheckerForTopRankedCompetitors) {
+    const singlePersonalBest = competitor.personalBests?.find(
+      personalBest =>
+        personalBest.eventId === eventId && personalBest.type === 'single'
+    );
+    const averagePersonalBest = competitor.personalBests?.find(
+      personalBest =>
+        personalBest.eventId === eventId && personalBest.type === 'average'
+    );
+    if (
+      (singlePersonalBest &&
+        (singlePersonalBest.worldRanking <= 100 ||
+          singlePersonalBest.nationalRanking <= 25)) ||
+      (averagePersonalBest &&
+        (averagePersonalBest.worldRanking <= 50 ||
+          averagePersonalBest.nationalRanking <= 15))
+    ) {
+      printScrambleCheckerBox = true;
```

Comment:
Now that it's a function you can just return the value, no need for separate variable:

```suggestion
      return true;
```